<ui version="4.0" >
 <class>content</class>
 <widget class="QWidget" name="content" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="IgnoreDesktopSettings" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>40</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Ignore the desktop settings (use this if everything is black)</string>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showServerBrowser" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>70</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Show the server browser on startup</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupTray" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>110</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title" >
    <string>Show icon in system tray</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <widget class="QCheckBox" name="doQuitOnClose" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Hide Esperanza on close instead of terminating it.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="singleOrDoubleClick" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>401</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Use double click on trayicon to show mainwindow.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
